application/src/main/java/org/thingsboard/server/service/entitiy/DefaultTbNotificationEntityService.java

```diff
@@ -78,14 +82,39 @@ public <E extends HasName, I extends EntityId> void notifyDeleteEntity(TenantId
     }
 
     public void notifyDeleteAlarm(TenantId tenantId, Alarm alarm, EntityId originatorId,
-                                  CustomerId customerId, ActionType actionType,
+                                  CustomerId customerId,
                                   List<EdgeId> relatedEdgeIds,
                                   SecurityUser user,
                                   String body, Object... additionalInfo) {
-        logEntityAction(tenantId, originatorId, alarm, customerId, actionType, user, additionalInfo);
+        logEntityAction(tenantId, originatorId, alarm, customerId, ActionType.DELETED, user, additionalInfo);
         sendAlarmDeleteNotificationMsg(tenantId, alarm, relatedEdgeIds, body);
     }
 
+    @Override
+    public void notifyDeleteRuleChane(TenantId tenantId, RuleChain ruleChain,
```

Comment:
**notifyDeleteRuleChane** typo - should be **notifyDeleteRuleChain**

Reply:
Ok